source/slang/slang-emit-spirv.cpp

```diff
@@ -579,14 +636,14 @@ struct SPIRVEmitContext
         IRInst* irInst = nullptr;
     };
 
-        /// Emit operand words for all the operands of a given IR instruction
-    void emitOperand(SpvInst* dst, OperandsOf const& other)
+        /// Emit operand words for all the operands of the current IR instruction
```

Comment:
The change to the comment here is incorrect.

The code change means that the operand words are emitted *to* the implicit destination instruction instead of the one given explicitly.

But the operands that get emitted are still those *of* the IR instruction that is given as `other`, and that is what the comment is referring to.